no-computer.md

```diff
@@ -11,41 +11,74 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I believe Lego can be a DSL. The need for Lego is met by people's desires to
+build something of their own design with lego blocks. This can be thought of 
+as running a program (the end product) with pre-defined functions (the lego 
+blcoks).
```

Comment:
I totally agree! When explaining to family and friends what this course is about, I discuss traditional DSLs and then also bring up an example of Legos.